Testing/VelaptorTests/Content/Caching/TextureCacheTests.cs

```diff
@@ -253,6 +253,8 @@ public void Ctor_WithNullReactableFactoryParam_ThrowsException()
     public void TotalCachedItems_WhenGettingValue_ReturnsCorrectResult()
     {
         // Arrange
+        this.mockPath.Setup(m => m.IsPathRooted(It.IsAny<string?>())).Returns(true);
+        this.mockPath.Setup(m => m.IsPathRooted(It.IsAny<string>())).Returns(true);
```

Comment:
These lines are duplicated in several locations in the file.  Remove one of the duplicates.